### PR TITLE
Add a setup.py to generate the script

### PR DESCRIPTION
**Pull request by rodfersou**
this way we can install easily with pipx:

```
pipx install git+https://github.com/rodfersou/swagger_to_uml.git
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+]
+
+setup(
+    name='swagger_to_uml',
+    version='0.1',
+    description='swagger_to_uml',
+    classifiers=[
+        "Programming Language :: Python"
+    ],
+    author='',
```
Comment (nlohmann) [inline setup.py:14]:
```suggestion
    author='Niels Lohmann',
```

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    author_email='',
```
Comment (nlohmann) [inline setup.py:15]:
```suggestion
    author_email='mail@nlohmann.me',
```

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    url='',
```
Comment (nlohmann) [inline setup.py:16]:
```suggestion
    url='http://nlohmann.me',
```

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    keywords='swagger uml',
```
Comment (nlohmann) [inline setup.py:17]:
```suggestion
    keywords='swagger uml plantuml',
```

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    packages=find_packages(),
+    include_package_data=True,
+    zip_safe=False,
+    install_requires=requires,
+    tests_require=requires,
+    scripts=['bin/swagger_to_uml']
+)
```
Comment (nlohmann) [inline setup.py:24]: Pllease also add

```py
license='MIT'
```

**Review by nlohmann — CHANGES_REQUESTED** [review]

File: setup.py (hunk 5 above)
Comment (rodfersou) [reply setup.py:25]: done

**Comment by rodfersou** [comment]
done

**Comment by rodfersou** [comment]
@nlohmann any reason for not merge this branch?

**Comment by nlohmann** [comment]
There is still an open suggestion regarding the email.

**Comment by rodfersou** [comment]
I see, sorry, didn't see that, done!

**Review by nlohmann — APPROVED** [review]
Looks good to me.

**Comment by nlohmann** [comment]
Thanks!

**Pull request merged**